src/pages/index.js

```diff
@@ -89,24 +153,95 @@ function closeModal(modal) {
   document.removeEventListener("keydown", handleEscClose);
 }
 
+function handleAvatarSubmit(evt) {
+  evt.preventDefault();
+  const submitButton = evt.submitter;
+  submitButton.textContent = "Saving...";
+  const profileImage = document.querySelector("#profile__image");
+api
+  .editAvatarInfo({ avatar: avatarImageInput.value })
+  .then((data) => {
+    profileImage.src = data.avatar;
+    closeModal(avatarModal);
+    evt.target.reset();
```

Comment:
* One remark hasn't been corrected https://github.com/AnnieM101/se_project_spots/pull/1#discussion_r2152444136 

Please, disable the `avatar` submit button after submission as well


```js

    evt.target.reset();
    disableButton(submitButton, config);  // this way
``